src/components/ExecutePassButton.js

```diff
@@ -1,14 +1,9 @@
 import React from 'react';
+import { useGameContext } from '../context/GameContext';
 import Card from './Card';
 
-export default function ExecutePassButton({
-  passCard,
-  setFrom,
-  from,
-  to,
-  selectedCard,
-  setSelectedCard,
-}) {
+export default function ExecutePassButton({ passCard }) {
+  const { selectedCard, setSelectedCard, from, setFrom, to } = useGameContext();
```

Comment:
here you will want to pass in multiple props because this file should hold the logic of passCare and findCareIndex. This logic was originally in App.js